src/modules/i18n/index.js

```diff
@@ -6,27 +6,23 @@ import fr from './locales/fr.json'
 import it from './locales/it.json'
 import rm from './locales/rm.json'
 
-export const languages = {
-    en: en,
-    de: de,
-    fr: fr,
-    it: it,
-    rm: rm,
-}
+export const languages = { en, de, fr, it, rm }
 
-// detecting navigator's locale as the default language (if it is a language served by this app)
-let matchingLang = null
+// detecting navigator's locale as the default language
+// (if it is a language served by this app)
+let matchedLanguage = null
 if (navigator.languages) {
-    navigator.languages.forEach((lang) => {
-        // we keep the first match we found
-        if (!matchingLang && lang in languages) {
-            matchingLang = lang
+    for (const language in navigator.languages) {
+        if (language in languages) {
+            matchedLanguage = language
+            // we keep the first match we found
+            break
         }
-    })
+    }
```

Comment:
I would maybe simplify that to
```
let matchedLanguage = null
if (navigator.languages) {
    // we keep the first match we found
    matchedLanguage = navigator.languages.find((lang) => lang in languages)
}
```
What do you think of that?

Reply:
I like! :sunglasses: